<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>996</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Моделирование движения в центральном поле сил</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
        <item>
         <layout class="QVBoxLayout" name="LayoutOfInitData">
          <item>
           <widget class="QGroupBox" name="GroupBoxInitialData">
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="title">
             <string>Ввод начальных данных</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="LabelEditWeight">
                 <property name="text">
                  <string>Масса, кг:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="EditWeight">
                 <property name="value">
                  <double>0.900000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="LabelModuleOfGravitysForce">
                 <property name="text">
                  <string>Модуль силы притяжения, Н:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="EditForce">
                 <property name="value">
                  <double>3.600000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="LabelInitialX0">
                 <property name="text">
                  <string>Координата x0, м:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="EditX0">
                 <property name="minimum">
                  <double>-99.989999999999995</double>
                 </property>
                 <property name="value">
                  <double>1.800000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="LabelInitialY0">
                 <property name="text">
                  <string>Координата y0, м:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QDoubleSpinBox" name="EditY0">
                 <property name="minimum">
                  <double>-99.989999999999995</double>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="LabelVx0">
                 <property name="text">
                  <string>Начальная скорость Vx0, м/с:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QDoubleSpinBox" name="EditVx0">
                 <property name="minimum">
                  <double>-99.989999999999995</double>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="LabelVy0">
                 <property name="text">
                  <string>начальная скорость Vy0, м/с:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QDoubleSpinBox" name="EditVy0">
                 <property name="minimum">
                  <double>-99.989999999999995</double>
                 </property>
                 <property name="value">
                  <double>2.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="LabelTime">
                 <property name="text">
                  <string>Время моделирования, с:</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QDoubleSpinBox" name="EditTime">
                 <property name="value">
                  <double>12.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="LabelIntegrationStep">
                 <property name="text">
                  <string>Шаг интегрирования, с:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QDoubleSpinBox" name="EditStepOfIntegration">
                 <property name="value">
                  <double>0.050000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Выпонлять в фоновом режиме</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Метод интегрирования:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonStart">
            <property name="text">
             <string>Смоделировать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonStopCalculation">
            <property name="text">
             <string>Прервать расчёт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonNewCalculation">
            <property name="text">
             <string>Новый расчёт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonExit">
            <property name="text">
             <string>Закрыть</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="GroupBoxResults">
          <property name="title">
           <string>Результаты</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>945</width>
                <height>469</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QTableView" name="Table">
                 <property name="alternatingRowColors">
                  <bool>false</bool>
                 </property>
                 <property name="showGrid">
                  <bool>true</bool>
                 </property>
                 <attribute name="horizontalHeaderCascadingSectionResizes">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
        <item>
         <widget class="QGroupBox" name="GroupBoxLResultList">
          <property name="title">
           <string>Список результатов расчётов </string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QListWidget" name="ListWidgetResults"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="ButtonActivateResult">
               <property name="text">
                <string>Открыть</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ButtonDeleteResult">
               <property name="text">
                <string>Удалить</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="GroupBoxGraphics">
          <property name="title">
           <string>Графики</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
